cpp/src/binaryop/jit/code/operation.cpp

```diff
@@ -400,6 +400,60 @@ const char* operation =
             return LogBase::operate<TypeOut, TypeLhs,TypeRhs>(y, x);
         }
     };
+
+    struct PMod {
```

Comment:
You could name this the complete name: `PositiveRemainder`

Reply:
done.

please relook, as i have removed some specializations and tried to use `std::fmod` as much as possible.

Reply:
Yep, still fine.